tests/test_application.coffee

```diff
@@ -87,20 +106,42 @@ Space.Error.extend CustomerApp, 'InvalidCustomerName', {
 
 class CustomerApp.Customer extends Space.eventSourcing.Aggregate
 
+  dependencies: {
+    myCommandDependency: 'myCommandDependency'
+    myEventDependency: 'myEventDependency'
+  }
+
   fields: {
     name: String
   }
 
   commandMap: -> {
     'CustomerApp.CreateCustomer': (command) ->
+      @myCommandDependency()
```

Comment:
The naming of this confuses me a little as it's a dependency on the Aggregate, not the message. Maybe `myAggregateDependency` would be more expressive